app/main.py

```diff
@@ -1,3 +1,61 @@
 class Distance:
     # Write your code here
-    pass
+    def __init__(self, km: float):
```

Comment:
add annotations for functions

Reply:
still, code needs annotations. for example in this one it should look like this:
 def __init__(self, km: int | float) -> None: